apps/lcs/sections/summary-data-sections.js

```diff
@@ -93,7 +93,14 @@ module.exports = {
       },
       {
         step: '/landlord-information',
-        field: 'rental-property-postcode'
+        field: 'rental-property-postcode',
+        parse: (value, req) => {
+          if (req.sessionModel.get('isCurrentTenant') ||
+            !req.sessionModel.get('steps').includes('/landlord-information')) {
+            return null;
+          }
```

Comment:
there is no need for a second check as `rental-property-postcode` should always be hidden if `isCurrentTenant` is true, the second check is irrelevant as the `/landlord-information` step is unavoidable